cmd/gaia/app/app.go

```diff
@@ -67,11 +70,14 @@ type GaiaApp struct {
 	mintKeeper          mint.Keeper
 	distrKeeper         distr.Keeper
 	govKeeper           gov.Keeper
+	crisisKeeper        crisis.Keeper
 	paramsKeeper        params.Keeper
 }
 
 // NewGaiaApp returns a reference to an initialized GaiaApp.
-func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bool, baseAppOptions ...func(*bam.BaseApp)) *GaiaApp {
+func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest, assertInvariantsBlockly bool,
```

Comment:
Also, this function signature is starting to get very lengthy. This is precisely why we added `baseAppOptions`. Why not just pass an option function `setAssertInvariantsBlockly(true|false)`?

Reply:
yeah I agree, unfortunately, this is _not_ a baseapp option but a gaia option... we could consolidate `loadLatest` and `assertInvariantsBlockly bool` into a gaia-options struct but I'm not sure that's necessary for the time being

Reply:
ahhhhh right! Darn it.